metadata-ingestion/tests/integration/cassandra/setup/cassandra.yaml

```diff
@@ -0,0 +1,229 @@
+cluster_name: 'Test Cluster'
```

Comment:
Please, let's keep the comment block here mentioning what is different in this Cassandra yaml as opposed to the one that comes with docker out of the box. It will be useful for future modifications.

Reply:
Its really fine for this file to be same as it was earlier (with initial code block plus rest of comments). The comment's intention was if we can selectively override a few configs by specifying only those instead of keeping this entire file.